sql-select/all-genres.sql

```diff
@@ -0,0 +1,3 @@
+select *
+  from "genres"
+  where "updatedAt" = '2007-02-15T09:46:27Z'
```

Comment:
For this one you want _all_ genres, not just the one that was updated at '2007-02-15T09:46:27Z'.